### PR TITLE
added closebtn and removed some nav styles

### PR DESCRIPTION
**Pull request by arshdkhn1**
* Removed rounded border for nav elements
* Sidebar now doesn't closes when click is inside navbar
* Close Buttton can be used to close sidebar

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32795

File: src/styles/main.scss [hunk 1]
```diff
@@ -198,3 +198,10 @@ li em{
     padding: 5px 15px;
   }
 }
+
+// For nav elements that have border-radius and left/right border
+.removeNavBorderStyles {
```
Comment (naveed-ahmad) [inline src/styles/main.scss]: we're not following camelCase convention for css, `remove-nav-border-styles` or `no-nav-border` would be much better name.

**Comment by mmahalwy** [comment]
Heads up, Globalsidebar is about to be deprecated in favor of `quran-components` drawer.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32803

File: src/styles/main.scss (hunk 1 above)
Comment (arshdkhn1) [inline src/styles/main.scss]: Ohk gotcha.

**Comment by arshdkhn1** [comment]
Ohk.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32807

**Pull request closed without merging**